packages/gasket-fetch/test/browser.spec.js

```diff
@@ -0,0 +1,12 @@
+import 'whatwg-fetch';
+import { describe, it } from 'mocha';
+
+import fetch, { AbortController, Request, Headers, Response } from '../browser';
```

Comment:
To be consistent, destructure these from the imported `fetch` like you did for the server.spec.js
```suggestion
import fetch from '../browser';
const { AbortController, Request, Headers, Response } = fetch;
```